src/i18n/locales/en.json

```diff
@@ -151,7 +151,7 @@
     "preorder items will be released to the warehouse you have selected.": "{count} preorder items will be released to the warehouse you have selected.",
     "preorders will be cancelled. This action cannot be undone.": "{count} preorders will be cancelled. This action cannot be undone.",
     "Products": "Products",
-    "Product details": "Product details",
+    "Product audit": "Product audit",
     "Product summary": "Product summary",
```

Comment:
We should remove this entry and re-add the '"Product details' entry as we are using it on the product details page.